src/main/java/game/ObjectManager.java

```diff
@@ -166,4 +172,15 @@ public PlayerState modifyPlayerState(Point2D playerPosition, PlayerState playerS
         }
         return playerState;
     }
+
+    public ArrayList<Element> getPortals() {
```

Comment:
Related to my comment on the code in the Game.java class. You don't need this method with the changes I suggested.